docs/content/guides/rows/rows-sorting.md

```diff
@@ -0,0 +1,3432 @@
+  columnSorting: true,
+});
+
+// disable sorting for all columns
+hotTableComponentRef.current.hotInstance.updateSettings({
+  columnSorting: false,
+});
+```
+
+:::
+
+You can also enable or disable sorting for specific columns.
+
+::: only-for javascript
+
+```js
+handsontableInstance.updateSettings({
+  columns: [
+    {
+      // enable sorting for the first column
+      columnSorting: {
+        headerAction: true,
+      },
+    },
+    {
+      // disable sorting for the second column
+      columnSorting: {
+        headerAction: false,
+      },
+    },
+  ],
+});
+```
+
+:::
+
+::: only-for react
+
+```jsx
+const hotTableComponentRef = useRef(null);
+
+hotTableComponentRef.current.hotInstance.updateSettings({
+  columns: [
+    {
+      // enable sorting for the first column
+      columnSorting: {
+        headerAction: true,
+      },
+    },
+    {
+      // disable sorting for the second column
+      columnSorting: {
+        headerAction: false,
+      },
+    },
+  ],
+});
+```
+
+:::
+
+### Sort data programmatically
+
+To sort data programmatically, use the [`columnSorting.sort()`](@/api/columnSorting.md#sort) method.
+Remember to [enable sorting](#enable-sorting) first.
+
+Mind that using [`columnSorting.sort()`](@/api/columnSorting.md#sort) overwrites any previous sort orders.
+
+::: only-for javascript
+
+```js
+// get the `ColumnSorting` plugin instance
+const columnSortingPluginInstance = handsontableInstance.getPlugin('columnSorting');
+
+columnSortingPluginInstance.sort(
+  // sort data by the first column, in ascending order
+  {
+    column: 0,
+    sortOrder: 'asc', // for descending order, use `'desc'`
+  },
+);
+
+// go back to the original order
+columnSortingPluginInstance.clearSort();
+```
+
+:::
+
+::: only-for react
+
+```jsx
+const hotTableComponentRef = useRef(null);
+
+// get the `ColumnSorting` plugin instance
+const columnSortingPluginInstance = hotTableComponentRef.current.hotInstance.getPlugin('columnSorting');
+
+columnSortingPluginInstance.sort(
+  // sort data by the first column, in ascending order
+  {
+    column: 0,
+    sortOrder: 'asc', // for descending order, use `'desc'`
+  },
+);
+
+// go back to the original order
+columnSortingPluginInstance.clearSort();
+```
+
+:::
+
+Try it out:
+
+::: only-for javascript
+
+::: example #example8 --html 1 --js 2
+
+```html
+<div id="example8"></div>
+
+<div class="controls">
+  <button id="sort_asc" class="button">Sort by the "Brand" column, in ascending order</button>
+  <br>
+  <br>
+  <button id="unsort" class="button">Go back to the original order</button>
+</div>
+```
+
+```js
+import Handsontable from 'handsontable';
+import 'handsontable/dist/handsontable.full.min.css';
+
+const container = document.querySelector('#example8');
+
+const button_sort_ascending = document.querySelector('#sort_asc');
+const button_unsort = document.querySelector('#unsort');
+
+
+const handsontableInstance = new Handsontable(container, {
```

Comment:
```suggestion
const buttonSortAscending = document.querySelector('#sort_asc');
const buttonUnsort = document.querySelector('#unsort');
const handsontableInstance = new Handsontable(container, {
```
I suggest using camelCase variable names for consistency with other examples within Docs.